gem/styles/gem-rwanda/modules/_m-profiles.scss

```diff
@@ -1,6 +1,21 @@
 .profiles {
   &__form {
     padding: 0 $spacer-mid;
+    & input {
+      border-radius: 0px;
+    }
+    & label {
+      text-transform: capitalize;
+      font-weight: bold;
+    }
+    & ul.errorlist, & span.error {
```

Comment:
@KaitCrawford Look great! Can you please add the pink or pink dark color for validation messages and remove padding on the left.